boards/arm/frdm_kw41z/frdm_kw41z.dts

```diff
@@ -49,15 +49,15 @@
 	pwmleds {
 		compatible = "pwm-leds";
 		blue_pwm_led: pwm_led_0 {
-			pwms = <&tpm2 0 0 0>;
+			pwms = <&tpm2 0 PWM_MSEC(20) 0>;
 			label = "User PWM LD1";
 		};
 		green_pwm_led: pwm_led_1 {
-			pwms = <&tpm2 1 0 0>;
+			pwms = <&tpm2 1 PWM_MSEC(20) 0>;
 			label = "User PWM LD2";
 		};
 		red_pwm_led: pwm_led_2 {
-			pwms = <&tpm0 2 0 0>;
+			pwms = <&tpm0 2 PWM_MSEC(20) 0>;
```

Comment:
Shouldn't also the `flags` cells be corrected to `PWM_POLARITY_INVERTED`? All these LEDs are active low according to the `leds` node above.

Reply:
right, good catch! fixed